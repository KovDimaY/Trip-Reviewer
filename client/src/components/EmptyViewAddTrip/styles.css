.empty-view-container {
  padding: 25px 0 50px 0;
  text-align: center;
}
.empty-view-container .message {
  margin-bottom: 75px;
  font-size: 24px;
}
.empty-view-container .button {
  font-size: 20px;
  padding: 10px 20px;
  background: #02bcd4;
  color: #fff;
  font-weight: 300;
  border-radius: 3px;
  border: none;
  margin-top: 50px;
}

@media (max-width: 400px) {
  .empty-view-container .message {
    font-size: 18px;
  }
}