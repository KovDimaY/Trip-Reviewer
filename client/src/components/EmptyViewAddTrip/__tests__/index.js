import React from 'react';
import { create } from 'react-test-renderer';

import EmptyViewAddTrip from '..';

const mockComponent = props => (
  <EmptyViewAddTrip {...props} />
);

describe('<EmptyViewAddTrip />', () => {
  it('should render component', () => {
    const tree = create(mockComponent()).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
