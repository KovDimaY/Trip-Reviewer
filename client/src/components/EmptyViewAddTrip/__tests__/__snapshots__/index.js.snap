// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EmptyViewAddTrip /> should render component 1`] = `
<div
  className="empty-view-container"
>
  <p
    className="message"
  >
    There is no any story yet.
  </p>
  <a
    className="button"
    href="/trip/add"
  >
    Add a story
  </a>
</div>
`;
